src/App.js

```diff
@@ -1,7 +1,25 @@
+import Header from "./Components/Header";
+import NavBar from "./Components/NavBar";
+import Articles from "./Components/Articles";
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Header>
```

Comment:
Not too sure why we have so many Headers here! If the outer tags are to be more semantic, move them into your Header component and return them from there